app/models/tag.rb

```diff
@@ -4,6 +4,8 @@ class Tag < ApplicationRecord
   virtual_has_one :category,       :class_name => "Classification"
   virtual_has_one :categorization, :class_name => "Hash"
 
+  has_one :container_label_tag_mapping # Only for tags assigned by ContainerLabelTagMapping.
```

Comment:
@cben Can you please explain what this relationship is for? I can't understand it from the pr